File: Algoritmos/Fibonacci/fibonacci.js
function fibonacci (num){
    let termo = +num
    let ultimo = 0
    let penultimo = 1

    for(let i = 1; i <= termo;i++){
        let atual = ultimo + penultimo
        console.log(atual)
        penultimo = ultimo
        ultimo = atual
    }
}
let a = lines.shift()
fibonacci(a)

File: Algoritmos/Fibonacci/fibonacci.py
def fibbonaci(num):
    termo = int(num)
    ultimo = 0
    penultimo = 1

    for i in range(1, termo+1):
        atual = ultimo + penultimo
        print(f"{atual}")
        penultimo = ultimo
        ultimo = atual

a = int(input())
fibbonaci(a)